api_tests/src/shared.ts

```diff
@@ -738,6 +740,17 @@ export async function listCommentReports(
   return api.client.listCommentReports(form);
 }
 
+export function getPosts(
+  api: API,
+  moderator_view = false
+): Promise<GetPostsResponse> {
+  let form: GetPosts | { moderator_view: boolean } = {
```

Comment:
I was planning to open a PR on the js client for exactly that anyway but if you can to quickly add it, it would only be the parameter moderator_view with type boolean?, The default is false.

Reply:
@biosfood Okay I've deployed a new lemmy-js-client : `0.18.3-rc.1`